pkgs/tools/misc/dua/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, darwin
+, rustPlatform, fetchFromGitHub
+, openssl, pkgconfig }:
```

Comment:
Please remove unused arguments.

Reply:
@killercup It looks like you don't need `stdenv`.

Reply:
Ah sorry, missed that one!